### PR TITLE
misc: add docs for `CYPRESS_SKIP_VERIFY` option

### PR DESCRIPTION
**Pull request by joshuajtward**
Adding the flag from [this PR](https://github.com/cypress-io/cypress/pull/29836/commits) to the table of env var options

### CODE REVIEW

**Review by jennifer-shehane — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
DO NOT MERGE UNTIL FEATURE IS MERGED

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**
